Workflow/Nodes/publishToHuggingFaceNode.ts

```
@@ -0,0 +1,512 @@
+
+  try {
+    const response = await fetch(url, {
+      method: "POST",
+      headers: {
+        Authorization: `Bearer ${token}`,
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        type: "dataset",
+        name: name,
+        organization: namespace === namespace ? undefined : namespace,
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)